core/src/free/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pref_cast_message">@string/pref_cast_message_free_flavor</string>
+</resources>
```

Comment:
I honestly don't know.
Github always complains with that red circle thingy and I submit to it out of bad conscience.